js/package.json

```diff
@@ -2,13 +2,13 @@
   "name": "surface",
   "version": "1.0.0",
   "description": "WIT IEEE Underwater ROV 2018 surface station",
-  "main": "ROV.js",
+  "main": "index.js",
   "scripts": {
     "test": "mocha test",
```

Comment:
How would I be able to access a camera feed from the ROV once we start testing everything in the water?

Reply:
We're probably gonna host a video stream on the pi. You'll get an address and a port.